salt/modules/swarm.py

```diff
@@ -30,10 +30,18 @@
 # Import Salt libs
 import salt.utils.json
 
+
+def _is_docker_module(mod):
+    if hasattr(mod, "APIClient") and hasattr(mod, "from_env"):
+        return True
+    else:
+        return False
```

Comment:
```suggestion
    required_attrs = ["APIClient", "from_env"]
    return all(hasattr(attr) for attr in required_attrs)
```
Pretty sure that's the right logic. Not a *necessary* change, but it makes it easier to add required_attrs in the future.